JamesRobertSutcliffe/personality-music-recommender
# Fix/non premium preview player display proper width

closes #55 

I believe I have found a solution to the play audio issue for non-premium users. I installed a package named "react-h5-audio-player" which seems to do the job. 

https://www.npmjs.com/package/react-h5-audio-player

To test this branch:

1 - npm i to install latest package
2 - Navigage to personalised homepage
3 - Click play icon and player should appear and autoplay selected song, you should be able to pause, skip and adjust volume in player. Should also be able to play other tracks once you hit another play icon. 

thanks! :)
